hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataReader.java

```diff
@@ -325,8 +325,10 @@ public OmKeyInfo lookupFile(OmKeyArgs args) throws IOException {
   public ListKeysResult listKeys(String volumeName, String bucketName,
       String startKey, String keyPrefix, int maxKeys) throws IOException {
     long startNanos = Time.monotonicNowNanos();
-    ResolvedBucket bucket = ozoneManager.resolveBucketLink(
-        Pair.of(volumeName, bucketName));
+    ResolvedBucket bucket = captureLatencyNs(
+            perfMetrics.getListKeysResolveBucketLatencyNs(),
+            () -> ozoneManager.resolveBucketLink(
+        Pair.of(volumeName, bucketName)));
```

Comment:
Indentation is a bit odd, but it can be fixed in a follow-up patch (whenever this file is being touched next time).

```suggestion
    ResolvedBucket bucket = captureLatencyNs(
        perfMetrics.getListKeysResolveBucketLatencyNs(),
        () -> ozoneManager.resolveBucketLink(
            Pair.of(volumeName, bucketName)));
```